hospital_list.json

```diff
@@ -0,0 +1 @@
网址httpwww.shtl120.com": {}, "上海同德医院": {}, "上海仁爱医院": {"电话": ["862164688888"], "网址": ["www.renai.cn"]}, "上海九州泌尿医院": {}, "上海玛丽女子医院": {"电话": ["862169726299"], "网址": ["www.mary120.cn"]}, "上海九龙男子医院": {"电话": ["862152739067"], "网址": ["www.long120.cn"]}, "上海长江医院": {"电话": ["862165022556"], "网址": ["www.cjhospital.com"]}, "上海真爱女子医院": {"证据": ["患者1httpwww.kobeei.comlc31713.htmlGoogle缓存httpwebcache.googleusercontent.comsearchsourceidchromepsyapi2ion1espv2ieUTF8qcache3A2F2Fwww.kobeei.com2Flc2F31713.htmloqcache3A2F2Fwww.kobeei.com2Flc2F31713.htmlrlz1C5CHFAenUS659US659aqschrome..69i57j69i58.4678j0j4"], "电话": ["862162269000"], "网址": ["www.shzhenai.com"]}, "上海玫瑰女子医院": {}, "上海阿波罗男子医院": {}, "上海新虹桥医院": {"电话": ["862162092255"], "网址": ["www.shhqyy.com"]}, "上海安真医院": {"证据": ["天涯广告httpbbs.tianya.cnpost417514551.shtmlGoogle缓存httpwebcache.googleusercontent.comsearchqcache3A2F2Fbbs.tianya.cn2Fpost417514551.shtmlrlz1C5CHFAenUS659US659oqcache3A2F2Fbbs.tianya.cn2Fpost417514551.shtmlaqschrome..69i57j69i58.2326j0j4sourceidchromeieUTF8"], "电话": ["862157711006"]}, "电话862165667633": {}, "上海曙康口腔医院": {}, "上海福华医院": {"电话": ["862151961999"]}, "上海心脏病医院": {"电话": ["862164829999"], "网址": ["www.yodak.net"]}, "江东泌尿外科医院": {}, "上海城市女子医院": {"电话": ["862151092999"]}, "上海国正医院注隶属上海明爱医疗集团": {}, "上海阳光中医医院": {}, "上海圣爱医院": {}, "浦东肝病专科医院": {}}, "吉安": {"江西吉安市第二人民医院": {}}, "伊宁": {"伊宁阳光女子医院": {}}, "绵阳": {"四川绵阳美康医院": {}}, "乌海": {"乌海现代医院": {}}, "丽江": {"丽江九洲医院": {}}, "九江": {"九江新世纪医院": {}, "九江阳光女子医院": {}, "九江玛丽亚医院": {}}, "男科": {"武汉现代泌尿外科医院": {"电话": ["862782886888"], "网址": ["www.xdmnyy.com"]}, "武汉名仕泌尿外科医院": {"电话": ["862788328080"], "网址": ["www.whmsnk.com"]}, "武汉阿波罗男科医院": {"电话": ["862785888669"], "网址": ["www.whablnk.com"]}, "武汉博大男科医院": {"电话": ["862785896666"], "网址": ["www.whnanke.net"]}, "武汉曙光中西医结合医院": {"电话": ["862788129788"]}}, "苏州": {"张家港朝阳五官科医院": {"电话": ["8651258983999"], "网址": ["www.cy5g.com"]}, "常熟东方妇科医院": {"电话": ["8651251531111"], "网址": ["www.csfk.net"]}, "常熟仁爱医院": {}, "苏州东吴医院": {"电话": ["8651268120120"], "网址": ["www.szdwyy.com"]}, "昆山阳光医院": {}, "苏州新医科": {}, "苏州华美美莱整形医院": {"电话": ["864008816499"], "网址": ["www.mlmryy.com"]}, "苏州圣爱医院": {"电话": ["8651268078047"], "网址": ["www.szsayy.com"]}, "昆山虹桥医院": {"电话": ["8651257399999"], "网址": ["www.ksyy120.com"]}}, "许昌": {"许昌凤凰医院": {}, "许昌中山医院": {"地址": ["许昌市西大街280号"], "网址": ["www.xczshp.com", "www.xcyyhp.com", "www.2158888.com", "www.xczsnk.com", "xczsnk.com"], "电话": ["03742158889"]}, "许昌新时代妇科医院": {}}, "唐山": {"唐山红十字医院": {}, "唐山女子医院": {}, "唐山京城皮肤病医院": {}}, "荆州": {"荆州华康医院": {}}, "杭州": {"杭州新东方妇产科医院": {}, "杭州虹桥医院": {"网址": ["www.hzhqyy.com"]}, "杭州天目山妇产医院": {"网址": ["www.97120.cn"]}, "杭州瑞丽医疗美容医院": {"网址": ["www.ruilizx.comwww.hzzxyy.ccwww.85181111.netwww.railyzx.com"]}, "杭州真爱医院": {}, "杭州建国医院": {}, "杭州广仁医院": {"网址": ["www.hzgryy.comwww.zjbybyw.com"]}, "杭州和睦医院": {"网址": ["www.hm120.cn"]}, "杭州阿波罗男子医院": {"网址": ["www.hzabl.comwww.hzabl.cn"]}, "杭州玛莉亚妇女医院": {"网址": ["www.hzmly.com"]}, "杭州同济医院": {"网址": ["www.0571tongji.comwww.hztjh.com"]}, "杭州萧山九龙男科医院": {"网址": ["www.hzjl120.comwww.nkxsjl.com"]}}}
```

Comment:
You probably want to keep this file away from root. The admin said he want to keep only one version of list to avoid confusion.